spec/spec.md

```diff
@@ -186,30 +186,34 @@ The following process defines how a DID-Relative URL for a Decentralized Web Nod
 **DID-Relative URL example for passing multiple messages:**
 
 ::: note
```

Comment:
Could you clarify here how this "DID-Relative URL example for passing multiple messages". I don't see multiple messages here. Am I missing something?

Reply:
Suggested fix, [above](https://github.com/decentralized-identity/decentralized-web-node/pull/228/files/dc69c89ffb12eb304f796f368f65b78e172d8826..a10509fab17347aa8db22cf39329fbf3670a9fcf#r1220696488).